enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -100,12 +100,7 @@ metadata:
   name: gateway
   namespace: openshift-ingress
 spec:
-  selector:                                                         # (1)
-    istio: ingressgateway
-  gatewayClassName: istio                                           # (2)
-  addresses:
-  - value: istio-ingressgateway.openshift-ingress.svc.cluster.local # (3)
-    type: Hostname
+  gatewayClassName: openshift-default
```

Comment:
I know that during conformance testing, we don't actually know the gateway is available until one of the `spec.addresses` responds.  It may be worthwhile in our endeavors to use the responsiveness of the gateway address to determine status.

Reply:
Do you mean the ingress operator could do something like the canary check but for the gateway?  That might become necessary in the far off future when replacing the default IngressController with a Gateway becomes feasible, but I don't think it's necessary at time time.

Reply:
That's fine.  I don't think the `addresses` field is an istio-ism though.

Reply:
"istio-ingressgateway.openshift-ingress.svc.cluster.local" has "istio" right in it.  * grin *.  Ideally the gateway owner should be able to put the desired name in `spec.addresses`, which is one thing I'm trying to achieve with this revision to the design.

Reply:
Moot.